src/main/java/com/synopsys/integration/alert/channel/jira/web/JiraCustomEndpoint.java

```diff
@@ -55,7 +55,7 @@
     private static final Logger logger = LoggerFactory.getLogger(JiraCustomEndpoint.class);
 
     // FIXME whenever we change the atlassian-connect.json file, we need to update this URL. This is very error prone and we'll want to find a better way to do this. Perhaps GH-pages
-    public static final String JIRA_PLUGIN_URL = "https://raw.githubusercontent.com/blackducksoftware/blackduck-alert/810dd02/src/main/resources/jira/atlassian-connect.json";
+    public static final String JIRA_PLUGIN_URL = "https://raw.githubusercontent.com/blackducksoftware/blackduck-alert/gk_jira_channel_enhancements/src/main/resources/jira/atlassian-connect.json";
```

Comment:
Do we need this URL? Can we have Alert host the file, instead of hosting it on github?

Reply:
This will eventually be hosted, this is just a workaround while we are developing.